molecule-runtime/src/commonMain/kotlin/app/cash/molecule/molecule.kt

```diff
@@ -130,8 +177,8 @@ public fun <T> CoroutineScope.launchMolecule(
  */
 public fun <T> CoroutineScope.launchMolecule(
   mode: RecompositionMode,
-  emitter: (value: T) -> Unit,
-  body: @Composable () -> T,
+  emitter: Emitter<T>,
+  body: @Composable (Emitter<T>) -> Unit,
```

Comment:
It's not clear why this function even exists in this new form. Why pass the emitter to the function only to propagate it into the body to be invoked? The body can reference the emitter directly at the callsite–Molecule doesn't need to be involved.

This seems more like instead you want a function which sets up an applier-less Composition in which you can do whatever you want. In reality you can do that already today with Molecule by using `Unit` as your type and an empty `emitter` lambda.
```kotlin
val myEmitter: (String) -> Unit = ::println

launchMolecule(emitter = {}) {
  myEmitter("Hello!")
}
```